File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/repository/jpa/CakeCategoryJpaRepository.java
package com.cakk.domain.mysql.repository.jpa;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;

import com.cakk.domain.mysql.entity.cake.CakeCategory;

public interface CakeCategoryJpaRepository extends JpaRepository<CakeCategory, Long> {

	Optional<CakeCategory> findByCakeId(Long cakeId);
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/repository/jpa/CakeCategoryJpaRepository.kt
package com.cakk.domain.mysql.repository.jpa

import com.cakk.domain.mysql.entity.cake.CakeCategory
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface CakeCategoryJpaRepository : JpaRepository<CakeCategory, Long> {

	fun findByCakeId(cakeId: Long?): CakeCategory?
}

